vunit/test/unit/test_test_report.py

```diff
@@ -161,7 +161,7 @@ def assert_has_test(self, root, name, time, status, output=None):  # pylint: dis
 
     def test_junit_report_with_all_passed_tests(self):
         report = self._report_with_all_passed_tests()
-        root = ElementTree.fromstring(report.to_junit_xml_str())
+        root = ElementTree.fromstring(report.to_junit_xml_str(xunit_xml_format='jenkins'))
```

Comment:
If you make 'jenkins' the default argument you can avoid this and similar lines.